src/config/dark-sites.config

```diff
@@ -313,6 +319,7 @@ hellofromhe.re
 hermes.ksdfg.me
 hidive.com
 himovies.to
+hltv.org
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/105616354-3217d180-5dce-11eb-8659-8b687ca64ffb.png)

Reply:
Their dark theme is based on the time zone, I'm not sure if you guys count this or not.
![2021-01-24 09_44_32-Window](https://user-images.githubusercontent.com/61620817/105623269-8603d400-5e29-11eb-8626-16936dbe0741.png)

Reply:
We don't count that.